### ISSUE TITLE
initial user karma does not allow posting links/images

### ISSUE THREAD

**Issue by berak** [opencv-infrastructure/answers.opencv.org#31]
with most noobs there, you just ask for more information, a helpful image, etc. but new users start with 0 karma, and 5 would be the least amount nessecary for posting a valid image or link.

while it makes some sense in the unmoderated case, to have users do a few things, to acquire karma first, imho it does not make any sense with the moderation situation, we're currently having.
(if they're "in", they're "in", no ?)

so, before asking for an image, we have to do like 5 clicks to add the nessecary karma, then add the comment, asking for the image , -- it's painful.

there seems to be no easy, official solution, looking at similar topics on the askbot forum, still, hacking this would clearly improve the QA site.

**Comment by StevenPuttemans** [comment]
It seems this is easy to solve, by editing the following file: https://github.com/ASKBOT/askbot-devel/blob/master/askbot/conf/minimum_reputation.py

I would suggest just to set the minimum rep for
- `MIN_REP_TO_ANSWER_OWN_QUESTION` = 0
- `MIN_REP_TO_ACCEPT_OWN_ANSWER` = 0
- `MIN_REP_TO_UPLOAD_FILES` = 0
- `MIN_REP_TO_SUGGEST_LINK` = 0
- `MIN_REP_TO_HAVE_STRONG_URL` = 0

Changing those will allow any accepted user to be able to do everything he needs.

**Comment by StevenPuttemans** [comment]
@mshabunin might it be possible to make a github repository for the code related to the forum? So we can make our own version with specific changes?

**Comment by StevenPuttemans** [comment]
Oh btw since we are busy, some other suggestions to change

- 'MAX_REP_GAIN_PER_USER_PER_DAY', default=200 --> given the fact that some users spend quite some time working on the forum, helping out others, I guess we should increase this level.
- 'MAX_VOTES_PER_USER_PER_DAY', default=30 --> same with this, as a mentor I get a limited feeling by this, because I want to acknowledge people's effort with an upvote. I vote increasing this.
- 'MIN_DAYS_FOR_STAFF_TO_ACCEPT_ANSWER', default=7 --> by now this results in tons of topics, finding a solution after only 1-2 days, which cannot be marked answered by moderators. We them tend to forget to head back and thus questions do not get marked solved. I would like to disable this.

These can all be found in the python configuration files.

**Comment by mshabunin** [comment]
I've disabled karma threshold for file upload and link posting. As for thresholds for answering own questions and accepting own answers, I think it will allow users to quickly increase their karma by posting questions and answering them, so I tend to leave those at 20 and 50.

Looks like Askbot stores all settings in a database, so we can not just put them to Git. However, I will try to make a repository with container config to ease site deployment.

I've increased max reputation gain to 400 and number of votes to 60. Minimum days to accept an answer was 2, I've changed it to 0.
